### PR TITLE
Add message to users with default nicks and prettify .dic output

### PR DESCRIPTION
**Pull request by hcolomb**
- Removed the notice to Pikmeir when a new user joins. 
- Added a message to users with default nicks from KiwiIRC to change their nicks. 
- Modified naver.py to use parenthesis around the number of each definition to make the output more readable.

### CODE REVIEW

File: join.py [hunk 1]
```diff
@@ -18,8 +18,10 @@ def on_join(connection, sender, chan):
...
     name = u'!'.join((identity.nick, identity.username))
     if not name in users:
-        connection.privmsg(chan, u'안녕하세요 {nick}님! {chan}에 처음 오신 것을 환영해요. Welcome to {chan}!'.format(chan=chan, nick=identity.nick))
-        connection.notice('Pikmeir', u'New visitor! XD')
-        users.append(name)
-        visitors._set(chan, users)
-        visitors._commit()
+        if name[:8] == 'ChangeMe':
+            connection.privmsg(chan, u'안녕하세요 {nick}님! Welcome to {chan}! You can change your name with "/nick new_name".'.format(chan=chan, nick=identity.nick))
+    	else:
```
Comment (youknowone) [inline join.py]: Here is indent mismatching. Can you fix it?

**Comment by hcolomb** [comment]
Fixed the spacing.

File: join.py [hunk 2]
```diff
@@ -18,8 +18,10 @@ def on_join(connection, sender, chan):
...
     if not name in users:
-        connection.privmsg(chan, u'안녕하세요 {nick}님! {chan}에 처음 오신 것을 환영해요. Welcome to {chan}!'.format(chan=chan, nick=identity.nick))
-        connection.notice('Pikmeir', u'New visitor! XD')
-        users.append(name)
-        visitors._set(chan, users)
-        visitors._commit()
+        if name[:8] == 'ChangeMe':
+            connection.privmsg(chan, u'안녕하세요 {nick}님! Welcome to {chan}! You can change your name with "/nick new_name".'.format(chan=chan, nick=identity.nick))
+        else:
+	    connection.privmsg(chan, u'안녕하세요 {nick}님! Welcome to {chan}!'.format(chan=chan, nick=identity.nick))
```
Comment (youknowone) [inline join.py]: Still. I suspect you are using hard tab(literal tab), instead of soft tab (spaces).

**Comment by hcolomb** [comment]
Not sure how those got there, but they're fixed. Went through both join.py and naver.py to make sure there were no more rogue tabs.

**Pull request merged**